initializr-service/src/main/resources/application.yml

```diff
@@ -1011,28 +1017,24 @@ initializr:
       content:
         - name: Cloud Bus
           id: cloud-bus
-          aliases:
-            - cloud-bus-amqp
-          description: A simple control bus defaulting to RabbitMQ
+          description: A simple control bus using Spring Cloud Stream (requires a binder, e.g. Kafka or RabbitMQ)
           groupId: org.springframework.cloud
-          artifactId: spring-cloud-starter-bus-amqp
+          artifactId: spring-cloud-bus
         - name: Cloud Stream
           id: cloud-stream
-          aliases:
-            - cloud-stream-binder-rabbit
-          description: Messaging microservices with Spring Cloud Stream and RabbitMQ
+          description: Messaging microservices with Spring Cloud Stream (requires a binder, e.g. Kafka or RabbitMQ)
           versionRange: 1.3.0.RELEASE
           groupId: org.springframework.cloud
-          artifactId: spring-cloud-starter-stream-rabbit
+          artifactId: spring-cloud-stream
         - name: Reactive Cloud Stream
           id: reactive-cloud-stream
-          description: Reactive messaging microservices with Spring Cloud Stream and RabbitMQ
+          description: Reactive messaging microservices with Spring Cloud Stream
           versionRange: 1.3.0.RELEASE
           groupId: org.springframework.cloud
-          artifactId: spring-cloud-starter-stream-rabbit
-        - name: Cloud Kafka Streams
+          artifactId: spring-cloud-stream-reactive
+        - name: Cloud Stream Kafka Streams
           id: cloud-stream-binder-kafka-streams
-          description: Stream processing microservices with Spring Cloud Stream and Kafka Streams API
+          description: Stream processing microservices with Spring Cloud Stream and Kafka Streams API (not needed for vanilla Spring Cloud Stream with Kafka)
```

Comment:
I know the "not needed for vanilla Spring Cloud Stream with Kafka" is supposed to help but it  confuses me. Why isn't that needed? Why do we have to state that?

Reply:
I want to prevent people from choosing this one when they don't know what it is, but they do know they want Streams + Kafka.